packages/sdk/src/fee/__tests__/FeeEstimator.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { OrditSDKError } from "../../errors";
+import { FeeEstimator } from "../FeeEstimator";
+import { createMockPsbt } from "../utils";
+
+describe("FeeEstimator", () => {
+  describe("constructor", () => {
```

Comment:
should we add test that `0` is actually valid feeRate by this logic?

Reply:
I'm not sure if 0 is a valid fee - @iamcrazycoder could you confirm?

Reply:
fee rate cannot be 0 and since Kevin has pointed that out, @Nanosync could you update the validation in constructor to `feeRate <= 0`? Thanks.

Reply:
but @iamcrazycoder  feeRate 0 i believe is used by sellerPSBTbuiler since seller dont pay for transaction.

Reply:
Added test for feeRate 0, along with a comment in the code indicating that this is intentional.

Reply:
Good catch! I almost forgot that lol

In that case, I think we need to think of a better approach to handle this case because we most certainly cannot allow 0 fee rate and proceed w/ considering it as normal by adding tests for it.